download/index.markdown

```diff
@@ -207,6 +207,7 @@ the filename need to be duplicated to `/etc/xbps.d/`.
   * https://youngjin.io/voidlinux (APAN: South Korea)
   * http://ftp.lysator.liu.se/pub/voidlinux (EU: Sweden)
   * http://lysator7eknrfl47rlyxvgeamrv7ucefgrrlhk7rouv3sna25asetwid.onion/pub/voidlinux (EU: Sweden)
+  * https://mirrors.hushan.tech:44300/voidlinux (Asia: China)
```

Comment:
Just double checking that this isn't a typographical error.  That's a high-order port which by convention we wouldn't expect to see services hosted on.

Reply:
You mean TCP 44300? Yes. It's hosted on non-commercial grade fiber. ISP's in China is under regulated and blocking 80, 8080 and 443 unless we "upgrade" it to commercial grade. With some luck, my home fiber (another ISP) has 443 unblocked but IPv6 only. Not sure when or if it would be blocked too so for now 44300 is my safest bet. I mean, if you don't mind it.

Reply:
Port number is fine, its just wildly non-standard so I wanted to double check.